Funcky.Test/TestUtils/EnumerateOnce.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections;
+using Xunit.Sdk;
+
+namespace Funcky.Test.TestUtils;
+
+public class EnumerateOnce<T> : IEnumerable<T>
+    where T : notnull
+{
+    private readonly Queue<T> _once;
+    private bool _first = true;
+
+    public EnumerateOnce(IEnumerable<T> sequence)
+        => _once = new Queue<T>(sequence);
+
+    public IEnumerator<T> GetEnumerator()
+    {
+        ValidateFirst();
+
+        while (true)
+        {
+            var maybeValue = _once.DequeueOrNone();
+
+            if (maybeValue.Match(false, True))
```

Comment:
Missing argument labels

Reply:
Fixed in 4e4e5096dced13412f81ce41dcaf54034837758e